apps/calendar/js/app.js

```diff
@@ -1,5 +1,6 @@
 define(function(require, exports) {
 'use strict';
+/* global IntlHelper */
```

Comment:
We usually put these declarations at the top-level (outside of the `define` block in calendar). Also, is there any way for calendar to bundle this via rjs instead of browser globals? We try to avoid globals where possible.